jack.py

```diff
@@ -92,6 +92,85 @@
     JackTransportStarting = 3,
     JackTransportNetStarting = 4,
 } jack_transport_state_t;
+typedef uint64_t jack_unique_t;
+typedef uint64_t jack_time_t;
+typedef uint32_t jack_nframes_t;
+typedef enum {
+
+    JackPositionBBT = 0x10,     /**< Bar, Beat, Tick */
```

Comment:
Please remove the documentation from the `cdef()` string.
The documentation should be below in the docstrings (which I think it already is).